### PR TITLE
Add Litmus

### PR DESCRIPTION
**Pull request by SteffenGivard**
Add [Litmus](https://www.litmus.com/) as limited.
There's an option to disassociate one's account from the email address used which they refer to as "simple deletion", however, I don't consider that a deletion.

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -10770,6 +10770,16 @@
         ]
     },
 
+    {
+        "name": "Litmus",
+        "url": "https://help.litmus.com/article/420-delete-your-account",
+        "difficulty": "limited",
+        "notes": "They offer a \"simple deletion\" where your account will be dissociated with your email address, and a [\"full data deletion\"](https://trust.litmus.com/?itemUid=af8eb7bc-d4b7-4e13-a4b7-6f05e2744c22) for complete data deletion.",
```
Comment (tupaschoal) [inline _data/sites.json:10777]: On the full data deletion do they verify that you're in a country with "deletion rights"?

Comment (SteffenGivard) [reply _data/sites.json:10777]: Nope, didn't have to provide anything.
Maybe they know based on IP from form submission or account usage 🤷

**Comment by SteffenGivard** [comment]
Sorry, been clicking all the wrong things 😅

File: _data/sites.json (hunk 1 above)
Comment (tupaschoal) [reply _data/sites.json:10777]: Likely if you choose a wrong country they don't check, but since we don't know, I'll keep it as `limited`

**Review by tupaschoal — APPROVED** [review]

**Pull request merged**